yamlkt/src/commonMain/kotlin/net.mamoe.yamlkt/YamlConfigurationInternal.kt

```diff
@@ -80,6 +86,42 @@ public class YamlBuilder internal constructor(
     @JvmField
     public var listSerialization: ListSerialization = conf.listSerialization
 
+    /**
+     * The suggested format for [Char] serialization.
+     *
+     * [Char] isn't always serialized in this format, depending on the content.
+     *
+     * Some escape sequences of special characters will be processed as escaped characters _(such as '\n')_
+     * */
+    public enum class CharSerialization {
+        /**
+         * Quote all [Char]s with `'`
+         *
+         * If a value can't be serialized using single quotation, it will use [CHAR_DOUBLE_QUOTATION]
+         */
+        CHAR_SINGLE_QUOTATION,
+
+        /**
+         * Quote all [Char]s with `"`
+         */
+        CHAR_DOUBLE_QUOTATION,
+
+        /**
+         * Encode [Char]s as their [code][Char.code] in integer.
+         * _(It will work like [Byte])_
+         *
+         * For example, the character 'A' will be converted to 65
+         */
+        CHAR_UNICODE_CODE,
```

Comment:
```suggestion
        SINGLE_QUOTATION,

        /**
         * Quote all [Char]s with `"`
         */
        DOUBLE_QUOTATION,

        /**
         * Encode [Char]s as their [code][Char.code] in integer.
         * _(It will work like [Byte])_
         *
         * For example, the character 'A' will be converted to 65
         */
        UNICODE_CODE,
```

As the enclosing class name is already prefixed with "Char" and the context is clear, I would prefer simpler enum entry names.

Reply:
Your concern makes sense. However, `CharSerialization.SINGLE_QUOTATION` is only a few characters longer than `CHAR_SINGLE_QUOTATION`, so it's not a big harm to not using `import *`. 

Another reason I don't like "CHAR_" is that it breaks consistency:

1. `StringSerialization.SINGLE_QUOTATION` and `CharSerialization. SINGLE_QUOTATION`
2. `StringSerialization.SINGLE_QUOTATION` and `CharSerialization. CHAR_SINGLE_QUOTATION`

The first looks better to me—the two names are consistent.